README.md

```diff
@@ -34,6 +34,15 @@ pnpm dev
 # http://localhost:3333
 ```
 
+## docker
+
+```bash
+# build
+docker build . -t henryclw/cook:dev
```

Comment:
In fact, I would prefer to use github actions to automatically deploy docker instead of under your account.

While this may seem like a lack of trust, I had to ensure docker ownership to avoid other issues.

Can you refer to here for modification? (If not, I can do it myself.)
> https://docs.docker.com/language/nodejs/configure-ci-cd/

Reply:
I've update the README in 691dd9bd . But I don't know your docker hub username, please change accordingly.

Reply:
Yeah, i had added it. My hub username is `yunyoujun`.

Reply:
Resolved in c6be3b7a